### PR TITLE
Add cypress-mailhog to Cypress dependencies

### PR DESCRIPTION
**Pull request by lukaszwnek**

### CODE REVIEW

**Review by mfly — APPROVED** [review]
@lukaszwnek it seems like I still have a write access to this repo. You may want to review the security settings. Nevertheless, LGTM :green_apple: :)

**Comment by lukaszwnek** [comment]
@mfly haha, I will definitely do that, thanks! happy Friday! 🍻

**Comment by mfly** [comment]
@lukaszwnek ok, I can't merge the PR so all good :). Happy Friday! :beers:

**Review by pawelrosa — APPROVED** [review]

**Pull request merged**
